docs/FAQs.md

```diff
@@ -57,8 +57,6 @@ When an organization contains a small amount of data, the backfilling process is
 
 - **Atlassian IP address ranges need whitelisting** - [Please see our IP allowlist documentation](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md)
 
-- **Reverse proxy support** - GitHub for Jira does not support reverse proxies.
-
```

Comment:
Should we add a link to https://support.atlassian.com/jira-cloud-administration/docs/connect-a-github-enterprise-server-account-to-jira-software/ here saying that we actually do support? :)

Reply:
Couldn't add a commit to this PR so created a different PR for that here: https://github.com/atlassian/github-for-jira/pull/2631